tests/neuralnet/test_keras.py

```diff
@@ -1,40 +1,28 @@
-import os.path
-
-import keras
-import pyomo.environ as pyo
+import tensorflow.keras as keras
 import pytest
+import pyomo.environ as pyo
 import tensorflow
```

Comment:
we don't have to import `tensorflow` and `tensorflow.keras`. we can modify this file to use either one

Reply:
fixed.